packages/suite/src/types/wallet/index.ts

```diff
@@ -19,11 +19,12 @@ import { FormDraftAction } from 'src/actions/wallet/formDraftActions';
 import { CardanoStakingAction } from 'src/actions/wallet/cardanoStakingActions';
 import { CoinjoinAccountAction } from 'src/actions/wallet/coinjoinAccountActions';
 import { CoinjoinClientAction } from 'src/actions/wallet/coinjoinClientActions';
-import { NETWORKS } from 'src/config/wallet';
 import { CoinmarketInfoAction } from 'src/actions/wallet/coinmarketInfoActions';
 import { tokenDefinitionsActions } from '@suite-common/token-definitions/src/tokenDefinitionsActions';
+import { networksCompatibility } from '@suite-common/wallet-config';
 
-export type Network = ArrayElement<typeof NETWORKS>;
+// TODO: why?
```

Comment:
Why indeed. The TS in this file was quite awkward.
Here is a side-PR with enhancement suggestion #13850 , feel free to fixup or ignore it :slightly_smiling_face: